packages/libs/error-stack/src/hook.rs

```diff
@@ -195,6 +264,73 @@ impl Report<()> {
     /// <pre>
     #[doc = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/tests/snapshots/doc/hook__fallback_builtin.snap"))]
     /// </pre>
+    ///
```

Comment:
Same as above

Reply:
fixed in [b303952](https://github.com/hashintel/hash/pull/974/commits/b303952afebd3291c6290ca50d77d2047854c773)